protocol/vdp_protocol.yaml

```diff
@@ -0,0 +1,166 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/instill-ai/vdp/blob/main/protocol/vdp_protocol.yaml
+title: VDP_Protocol
+type: object
+description: VDP Protocol structures
+properties:
+  batch_outputs:
+    "$ref": "#/definitions/BatchOutputs"
+definitions:
+  BatchOutputs:
+    type: array
+    items:
+      "$ref": "#/definitions/TaskOutput"
+  TaskOutput:
+    type: object
+    additionalProperties: true
+    minProperties: 2
+    required:
+      - task
+    properties:
+      task:
+        description: "Task type"
+        type: string
+        enum:
+          - CLASSIFICATION
+          - DETECTION
+          - KEYPOINT
+          - UNSPECIFIED
+      classification:
+        description: "Classify into pre-defined categories"
+        "$ref": "#/definitions/Classification"
+      detection:
+        description: "Detect and localize multiple objects"
+        "$ref": "#/definitions/Detection"
+      keypoint:
+        description: "Detect and localize keypoints of multiple objects"
+        "$ref": "#/definitions/Keypoint"
+      unspecified:
+        description: "Unspecified task with output in the free form"
+        "$ref": "#/definitions/Unspecified"
+  Classification:
+    type: object
+    additionalProperties: true
+    required:
+      - category
+      - score
+    properties:
+      category:
+        description: "The predicted category of the input"
+        type: string
+      score:
+        description: "The confidence score of the predicted category of the input"
+        type: number
+  Detection:
+    type: object
+    additionalProperties: true
+    required:
+      - bounding_boxes
```

Comment:
@xiaofei-du should we have no-required property `label` for objects?

Reply:
What does the `label` for? Is it the `category`?

Reply:
@xiaofei-du yes, it is same.